services/sessionservice/session.go

```diff
@@ -110,7 +110,7 @@ func (sessionService *sessionServiceImpl) add(session *models.Session) httperror
 }
 
 // Initialize the session service
-func (sessionService *sessionServiceImpl) init() error {
```

Comment:
Remove this changes from commit "fix typo in Authentication" because it's not a typo.